File: SoalB/C++/product.cpp
#include <string>
#include <iostream>

using namespace std;

class product
{
    /* product adalah parent dari semua class */

    /* atribut privat */
private:
    string idProduct;
    string price;

public:
    /* membuat constructor */
    product()
    {
    }

    /* Get-Set Methods idProduct */
    void setID(string idProduct)
    {
        this->idProduct = idProduct;
    }
    string getID()
    {
        return this->idProduct;
    }

    /* Get-Set Methods price */
    void setPrice(string price)
    {
        this->price = price;
    }
    string getPrice()
    {
        return this->price;
    }

    /* membuat destructor */
    ~product()
    {
    }
};

File: SoalB/PHP/Product.php
<?php

class Product
{
    /* product adalah parent dari semua class */

    /* atribut privat */
    private $price,
        $idProduct;

    /* membuat constructor */
    public function __construct()
    {
        $this->price = "";
        $this->idProduct = "";
    }

    /* Get-Set Methods price */
    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function getPrice()
    {
        return $this->price;
    }

    /* Get-Set Methods idProduct */
    public function setIdProduct($idProduct)
    {
        $this->idProduct = $idProduct;
    }

    public function getIdProduct()
    {
        return $this->idProduct;
    }

    public function __destruct()
    {
    }
}
